main.py

```diff
@@ -86,20 +86,13 @@ def create_box(row, fieldnames: list[str]) -> str:
         for row in reader:
             box_cmd += create_box(row, selected_fields)
 
-except FileNotFoundError:
-    print('Unable to find', CSV_FILE)
-
-try:
     final = ''
     with open(TEMPLATE_FILE, 'r', encoding='utf8') as template:
         for line in template:
             replaced = line.replace(PLACEHOLDER_CMD, cmd)
             replaced = replaced.replace(PLACEHOLDER_CCOMMENT, box_cmd)
             final += replaced
-
-    with open(TARGET_FILE, 'w', encoding='utf8') as target:
+    args.tex_file.mkdir(parents=True, exist_ok=True)
```

Comment:
This create a folder with the file name and causes an error in the next step:

`IsADirectoryError: [Errno 21] Is a directory: 'out.tex'`

Reply:
Sorry, fixed. :)